consensus/service/config/src/tokens.rs

```diff
@@ -912,29 +793,23 @@ mod tests {
         let minting_trust_root_private_key =
             Ed25519Private::try_from_der(&minting_trust_root_private_key_pem.contents[..]).unwrap();
 
-        let input_toml: &str = r#"
-            # Signature generated by taking the above file and the admin private key above and running:
-            # cargo run --bin mc-consensus-mint-client -- sign-governors --tokens tokens.toml --signing-key private.pem
-            governors_signature = "a07b628ebb74acd222a8d267f01dedbf1389db7ec3f2bb51d2270f4b43b4f2ebcdd5b43ee7574783ba483b03d9e2fb92dd0ed9d993509dad935532aede18790c"
-
-            [[tokens]]
-            token_id = 0 # Must have MOB
-
-            [[tokens]]
-            token_id = 1
-            minimum_fee = 1
-            [tokens.governors]
-            signers = """
-            -----BEGIN PUBLIC KEY-----
-            MCowBQYDK2VwAyEAyj6m0NRTlw/R28Q+R7vBakwybuaNFneKrvRVAYNp5WQ=
-            -----END PUBLIC KEY-----
-            -----BEGIN PUBLIC KEY-----
-            MCowBQYDK2VwAyEAl3XVo/DeiTjHn8dYQuEtBjQrEWNQSKpfzw3X9dewSVY=
-            -----END PUBLIC KEY-----
-            """
-            threshold = 2 # The signature was generated when this was set to 1
-       "#;
-        let tokens: TokensConfig = toml::from_str(input_toml).expect("failed parsing toml");
+        // Made invalid by changing the token id
+        // A previous test verified the signature is valid.
+        let input_json: &str = r#"{
+            "governors_signature": "d68fb53632835c23209528387cab9722bc5a2e6d092138468efa5babe11af7c2a20412cd90dcce2344febb23570e1961e6da37aa01e0bd9db4697a910f9fa408",
+            "tokens": [
+                { "token_id": 0 },
+                {
+                    "token_id": 1000,
+                    "minimum_fee": 2,
+                    "governors": {
+                        "signers": "-----BEGIN PUBLIC KEY-----\nMCowBQYDK2VwAyEAyj6m0NRTlw/R28Q+R7vBakwybuaNFneKrvRVAYNp5WQ=\n-----END PUBLIC KEY-----\n-----BEGIN PUBLIC KEY-----\nMCowBQYDK2VwAyEAl3XVo/DeiTjHn8dYQuEtBjQrEWNQSKpfzw3X9dewSVY=\n-----END PUBLIC KEY-----",
```

Comment:
:thought_balloon: I'm all for removing functionality if we don't think it will be used.
But this example here, no multiline strings, and JSON's lack of real comments have generally held it back from being good configuration formats.  If we're providing a viewer or similar that generates these then it may be a non issue.

While I know that nested structures in toml isn't as nice as it could be, https://stackoverflow.com/a/48998532 it still seems doable

Reply:
It has been confirmed that it's not used since our ops team prefers to use JSON.
I agree JSON's lack of multiline strings or comments is irritating and a downgrade from toml, but nothing is currently using the toml support anyway.